### PR TITLE
[IMPROVE] Adds tooltip for sidebar header icons

### LINKED ISSUE #19908: Tooltip for SideNav Header Icons
### Description:

It would be helpful to have a tooltip corresponding to the icons in the sidenav header similar to the ones displayed for all other icons within Rocket Chat. Some users might find the icons in the sidenav header ambiguous and may realise what it does only after clicking on it.

### Steps to reproduce:

1. Hover over the icons in the sidenav header.

### Expected behavior:

Tooltip to appear similar to the ones for the below shown icons.

Screenshot from 2020-12-19 22-29-44

### Actual behavior:

No tooltip for sidenav header icons.

Screenshot from 2020-12-19 22-30-02

### Server Setup Information:

- Version of Rocket.Chat Server: 3.9.3
- Operating System: Ubuntu
- Deployment Method:
- Number of Running Instances: 
- DB Replicaset Oplog: 
- NodeJS Version: 12.20.0
- MongoDB Version: 4.2.8

### Client Setup Information

- Desktop App or Browser Version: Google Chrome
- Operating System: Ubuntu 18.04

### PR DESCRIPTION
**Pull request by RonLek**
## Proposed changes (including videos or screenshots)

Previously the header icons in the sidebar didn't show a tooltip when hovered over. This PR fixes that.

Screenshot from 2020-12-22 15-17-41

## Issue(s)

Closes #19908

### CODE REVIEW

**Comment by RonLek** [comment]
@dougfabris do you think it would be a good idea to replace the sort icon with something more intuitive since it represents more than just sort (for example, view modes and group by)? Would love to hear your thoughts on this.

File: client/sidebar/header/index.js [hunk 1]
```diff
@@ -19,15 +19,15 @@ const HeaderWithData = () => {
...
+				<Home title='Home' />
+				<Search title='Search' data-qa='sidebar-search' />
 				{user && <>
-					<Directory />
-					<Sort />
-					<CreateRoom data-qa='sidebar-create' />
-					<Menu />
+					<Directory title='Directory' />
+					<Sort title='Sort' />
+					<CreateRoom title='Create' data-qa='sidebar-create' />
```
Comment (dougfabris) [inline client/sidebar/header/index.js:27]: I think it's better to use "Create Room"

**Review by dougfabris — CHANGES_REQUESTED** [review]
> @dougfabris do you think it would be a good idea to replace the sort icon with something more intuitive since it represents more than just sort (for example, view modes and group by)? Would love to hear your thoughts on this.

Well, for now, we shouldn't replace the icon. Just replace the title for **Filters**, I think it can be more descriptive.

**Comment by RonLek** [comment]
Thanks! @dougfabris. Made the changes.

**Comment by dougfabris** [comment]
@RonLek Thank you for helping us =)

**Review by dougfabris — APPROVED** [review]

**Review by ggazzo — CHANGES_REQUESTED** [review]
hey thanks for your time, can I ask you to use translations instead of the raw text? 

look for `useTranslation`

**Comment by RonLek** [comment]
@ggazzo Definitely! For the ones not already having translations in languages other than English should I add them in this PR itself or raise a different one similar to [this](https://github.com/RocketChat/Rocket.Chat/pull/19916)?

**Comment by dougfabris** [comment]
@RonLek as these translations are related to the fix I think it's no problem to add in this PR.

**Comment by RonLek** [comment]
Thanks! @dougfabris . Added translations in this PR itself : )

**Comment by RonLek** [comment]
@dougfabris @ggazzo do let me know if any changes required. 

Even though the translations are double-checked should translations be done only in English for the time being in this PR since we're taking in native speakers' contributions for translations? 

Awaiting your views on this one : )

**Comment by ggazzo** [comment]
@RonLek  I would ask you to do not create one more key, please use `Create_A_New_Channel` instead of `Create_Room `

**Comment by RonLek** [comment]
Thanks @ggazzo! I've made the changes  :)

**Comment by RonLek** [comment]
Thanks @ggazzo. I've made the changes. Please review.

**Comment by RonLek** [comment]
@dougfabris @ggazzo please take a look :)

**Comment by ggazzo** [comment]
@RonLek can I ask you from where did you get the translations? I'm not very fluent in some of them :p just to be sure that makes sense

**Comment by RonLek** [comment]
@ggazzo Definitely! All of the translations have been taken up from Google Translate after a double-check. 

What I mean by a double-check is that if "Home" translates to "Casa" in Spanish, then I've reverse translated "Casa" to English and seen if that checks out back to "Home" and only then inserted the translation for the word (skipped in cases it didn't). The reason for this is that I'm aware Google Translate might not capture the context of the word when just a single word is entered for translation but doing a reverse-check confirms if the translation was correct.

Let me know what you think : )

**Comment by ggazzo** [comment]
Ok, `casa` in portuguese means home as well but still I'm not sure if is a common usage for that word... but no problem thanks

**Pull request merged**